src/tests/ftest/harness/advanced.py

```diff
@@ -43,24 +45,21 @@ def test_core_files(self):
                 pid = match[0]
             index += 1
         if pid is None:
-            self.fail(
-                "Error obtaining pid of the daos_engine process on "
-                "{}".format(host))
+            self.fail("Error obtaining pid of the daos_engine process on {}".format(host))
         self.log.info("Found pid %s", pid)
 
         # Send a signal 6 to its daos_engine process
         self.log.info("Sending a signal 6 to %s", pid)
-        result = run_pcmd([host], "sudo kill -6 {}".format(pid))
+        result = run_pcmd(host, "sudo kill -6 {}".format(pid))
```

Comment:
(pylint-consider-using-f-string) Formatting a regular string which could be a f-string